src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search {
+  display: block;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.bigSearch__input {
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  margin-top: 20px;
+  padding-left: 62px;
+  background: url(images/Search.svg) no-repeat 26px 25px;
+}
+
+.bigSearch__input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
```

Comment:
You already declare for default input selector
font-family: "Avenir", sans-serif;
color: #3d4e61;
font-weight: 300;
font-size: 16px;